packages/backend/src/accounting/psql/service.test.ts

```diff
@@ -126,4 +130,199 @@ describe('Psql Accounting Service', (): void => {
       })
     })
   })
+
+  describe('getBalance', (): void => {
+    let account: LedgerAccount
+
+    beforeEach(async (): Promise<void> => {
+      account = await createLedgerAccount(
+        {
+          accountRef: asset.id,
+          ledger: asset.ledger,
+          type: LedgerAccountType.LIQUIDITY_INCOMING
+        },
+        knex
+      )
+    })
+
+    test('gets balance for existing account', async (): Promise<void> => {
+      await expect(
+        accountingService.getBalance(account.accountRef)
+      ).resolves.toBe(0n)
+    })
+
+    test('returns undefined for non-existing account', async (): Promise<void> => {
+      await expect(
+        accountingService.getBalance(uuid())
+      ).resolves.toBeUndefined()
+    })
+  })
+
+  describe('getTotalReceived', (): void => {
```

Comment:
for now, fairly basic tests, once transfers are flushed out I'll think of a way to add more comprehensive scenarios